### PR TITLE
Feature branch milestone.

### PR DESCRIPTION
**Pull request by Nowembabazi**
##In this milestone, we achieved the following.
Description the project.
-Added html tags to my page. 
-Used JavaScript to my page for more functionality.
-Used bootstrap for styling some aspects on my page. 
-Created a MIT.md file.
-Applied a background to make my page look better.
-Applied icons to my page.
-styled using CSS style to beautify my page.
-Applied linters to my project. 
-created a .gitingnore file.

[link] https://www.loom.com/share/5f79a3715f8d48299536b1f816162458..

### CODE REVIEW

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2 /
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
Comment (Bondok6) [inline .github/workflows/linter.yml]: - [ ] Kindly, for this project you should use [html-css-js](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) Linters 💡

**Review by Bondok6 — COMMENTED** [review]
## Hi @Nowembabazi 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have `eslint` Linters (Js Linter), kindly update `linters.yml` file to setup Linters correctly.
1. Kindly, use [html-css-js](https://github.com/microverseinc/linters-config/tree/master/html-css-js) Linter
1. Linter should be set up correctly for this repository.
1. Linter result should be green, this means that the last commit have a green check mark (✅)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            <a class="nav-items" href="#">News</a>
+            <a class="nav-items-2" href="#">dc Campaign</a>
+        </nav>
+        <nav class="nav-list">
+            <a  class="nav-items" href="./index.html">Home</a>
+            <a  class="nav-items" href="./about.html">About</a>
+            <a class="nav-items" href="#">Program</a>
+            <a class="nav-items" href="#">News</a>
+            <a class="nav-items-2" href="#">dc Campaign</a>
+        </nav>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Kindly make sure that You follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: Colors, Typography: font face, size and weight and Layout: composition and space between elements. The taller bottom bar with a logo and a list of page links should be white bar just like given design Template.
The Design Nav Bar Look likes this : 
sample

Your Project Nav Bar Look likes this : 
sample1

File: about.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+            <a class="nav-items" href="#">News</a>
+            <a class="nav-items-2" href="#">dc Campaign</a>
+        </nav>
+        <nav class="nav-list">
+            <a  class="nav-items" href="./index.html">Home</a>
+            <a  class="nav-items" href="./about.html">About</a>
+            <a class="nav-items" href="#">Program</a>
+            <a class="nav-items" href="#">News</a>
+            <a class="nav-items-2" href="#">dc Campaign</a>
+        </nav>
```
Comment (Abenezer-Tilahun) [inline about.html]: - [ ] Kindly make sure that You follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: Colors, Typography: font face, size and weight and Layout: composition and space between elements. The taller bottom bar with a logo and a list of page links should be white bar just like given design Template.
The Design Nav Bar Look likes this : 
sample

Your Project Nav Bar Look likes this : 
sample1

File: index.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    <header class="navbar">
+        <nav class="navbar-1">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a class="nav-items-11" href="#">English</a>
+            <a class="nav-items-12" href="#">My Page</a>
+            <a class="nav-items-13" href="#">Logout</a>
+        </nav>
+        <nav class="navbar-2">
+            <h2 class="nav-logo">SUAVE INTERIORS</h2>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] As provided in the project requirements for interaction, kindly make the logo in the header links to the home page.
cap1

File: index.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+        <div class="quote">
+            <p class="quote-1">We bring to you great speakers from all around the world to <br>
+                boast you knowlegdge and understanding of interior designs.<i class="fas fa-broadcast-tower    "></i>
+                 </p>
+        </div>
+        <div class="details">
+            <p class="details-1">2022.10.12(FRI) ~ 12(SAT)</p>
+               <p class="details-2"> @ lugogo Cricket oval.</p>
+        </div>
+    </section>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Kindly make sure that You follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: Colors, Layout: composition and space between elements.
- [ ] For class="title1" `h3` Kindly adjust color just like the design 
Capt1

Your project Design Look likes this : 
Captur12

- [ ] Kindly also adjust Layout: composition and space between elements for class="details" of second `p` it just overlapped by the second section just show above on image.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+               <li class="box-1">DC Ignite</li>
+               <p class="box-2">Join our webinars and live training sessions.</p>
+            </div>
+          </div>
+        </ul>
+        <a class="mp-1">SEE THE WHOLE PROGRAM</a>
+        <button class="button">Join Interior Symposium 2022</button>
+     </div>
+    </section>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Kindly  adjust Layout: composition and space between elements for section. 

Your Main program section Look Likes this : 
C2

The Design Main section Look Likes this : 
C1

File: script.js [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+  speakerHistory.textContent = speakerPara.history;
+  contentSection.appendChild(speakerHistory);
+
+  speakerSection.appendChild(contentSection);
+  speakerContainer.appendChild(speakerSection);
+});
+
+const body = document.querySelector('body');
+const partners = document.getElementById('partners');
+body.insertBefore(speakerContainer, partners);
```
Comment (Abenezer-Tilahun) [inline script.js]: - [ ] Good Job for Implementing  the section "Featured speakers"  dynamically in JavaScript. But still the pages should look almost identical to the original design. So Kindly Work on styling page just Like the Design. Adjust page background color, Underline layout position and speakers background image just like the design.

The Template design Look likes this : 
 
C1

Your Project Design for page Look Likes this : 
C2

File: about.html [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+            <div class="row d-flex justify-content-center">
+              <div class="col-4 d-flex justify-content-center">
+                 <h2 class="nav-logo">SUAVE INTERIORS</h2>
+              </div>
+              <div class="col-4 d-flex justify-content-center align-items-center">
+                <p>2022 Creative Commons Korea <br>
+                Some Rights Reserved </p>
+              </div>
+            </div>
+          </footer>
```
Comment (Abenezer-Tilahun) [inline about.html]: - [ ] Kindly make sure that the pages  look almost identical to the original design. For Desktop size In the About page there is no partner section just like design show below and for footer section Adjust  background color just like Design Template.

The Design About Page Look Likes this : 
 
C1

Your Project About Page Look Likes this : 
C2

File: index.html [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+            <h3 class="featured">Featured Speakers</h3>
+            <hr class="hr-title">
+          </div>
+        </div>
+          <div class="col-12 text-center mb-5 border d-block d-md-none">
+            <a data-toggle="collapse" data-target="#demo" href="#" class="text-uppercase btn rounded-0 pr-5 pl-5 pt-2 pb-2">
+              More <i class="fas fa-angle-down red"></i>
+            </a>
+          </div>
+      </section>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] Kindly make sure that the pages  look almost identical to the original design. For Mobile screen size In the feature speaker section design look different from the original design. Please try to adjust it.
Your Project Look likes this : 
C1

The Template design Look Like this : 

C2

File: about.html [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+            <div class="about-box">
+            <p class="about-box-2">The logo of this symposium was designed by Phillip Hall in 2016.</p>
+            <div class="logo-img">
+            <h2>SUAVE INTERIORS.</h2>
+            
+        </div>
+        
+        </ul>
+    </div>
+    </section>
```
Comment (Abenezer-Tilahun) [inline about.html]: - [ ] Kindly make sure that You follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), For Mobile screen size the text are overlapping please try to adjust Layout: composition and space between elements.

Your Project Look likes this : 
C1

The Design Template Look likes this : 
Ca1

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Nowembabazi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

## Required Changes ♻️
_Check the comments under the Review.

- [ ] Good Job for documenting your project in professional way , But try to make your video contains a clear and objective description about the project with  demo of the project, showing the pages in different screen sizes and finally a highlight about a piece of the code the your finds interesting or is proud of making.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (arslanbisharat) [reply index.html]: - [ ] This still needs to work. The design doesn't match your website. there should be no gap at the edges. 

image

File: about.html (hunk 3 above)
Comment (arslanbisharat) [reply about.html]: - [ ] The position of the text inside the box still needs to be aligned as requested by the previous CR. 
image

File: index.html (hunk 4 above)
Comment (arslanbisharat) [reply index.html]: Good Job. This looks good.

File: index.html (hunk 5 above)
Comment (arslanbisharat) [reply index.html]: - [ ] This still needs work to be done. The text should be aligned as requested by the previous CR.

File: index.html (hunk 6 above)
Comment (arslanbisharat) [reply index.html]: - [ ] This still needs work to be done. 

image

**Comment by arslanbisharat** [comment]
Hi @contributor_name, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments above the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+            <hr class="line hr-title">
+          </div>
+        </div>
+         
+      </section> 
+       <!-- <div class="col-12 text-center mb-5 border d-block d-md-none">
+            <a data-toggle="collapse" data-target="#demo" href="#" class="text-uppercase btn rounded-0 pr-5 pl-5 pt-2 pb-2">
+              More <i class="fas fa-angle-down red"></i>
+            </a>
+          </div> -->
```
Comment (emmiiorji) [inline index.html]: Kindly remove commented code from your codebase to make for more readability of the code.

File: script.js [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+  contentSection.appendChild(speakerName);
+
+  const speakerDesignation = document.createElement('p');
+  speakerDesignation.setAttribute('class', 'speaker-designation');
+  speakerDesignation.textContent = speakerPara.designation;
+  contentSection.appendChild(speakerDesignation);
+
+  const dotText = document.createElement('p');
+  dotText.setAttribute('id', 'dots');
+  dotText.textContent = '.........';
```
Comment (emmiiorji) [inline script.js]: As recommend by the previous reviewer, kindly adjust the speakers' section to match the given design. Make the dividing rule in the speaker cards to be like it is on the design. It's a continuous line and not dotted.

Also italicise and reduce the font size of the title of the speaker on red text and add the mini background behind the speakers' pictures as it appears on the design.



Your design:
image


Design guideline:
image

File: index.html [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+        </nav>
+        <nav class="navbar-2">
+          <a href="./index.html"><h2 class="nav-logo">SUAVE INTERIORS</h2></a>
+            <!-- <a class="nav-items" href="./index.html">Home</a> -->
+            <a class="nav-items" href="./about.html">About</a>
+            <a class="nav-items" href="#">Program</a>
+            <a class="nav-items" href="#">Join</a>
+            <a class="nav-items" href="#">News</a>
+            <a class="nav-items-2" href="#">dc Campaign</a>
+        </nav>
```
Comment (emmiiorji) [inline index.html]: [Optional] I recommend that you adjust the bar to match the design exactly. The light-coloured bar above the dark-coloured bar does not appear in the design guideline. Also notice how the `logout` should be vertically aligned with the red bordered box.

Your design:
image

Design guideline:
image

File: index.html [hunk 14]
```diff
@@ -0,0 +1,159 @@
...
+            <li class="box-1">Exhibition</li>
+              <p class="box-2">Join our webinars and live training sessions.</p>
```
Comment (emmiiorji) [inline index.html]: Good job at working on the text-overflow as recommended by the previous reviewer. Kindly adjust the padding within the cards as shown in the illustration to match the design.

Your design:
image

Design guideline:
image

File: about.html [hunk 15]
```diff
@@ -0,0 +1,153 @@
...
+<body>
+  <section class="main-section">
+    <header class="navbar">
+        <nav class="navbar-1">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a class="nav-items-11" href="#">English</a>
+            <a class="nav-items-12" href="#">My Page</a>
+            <a class="nav-items-13" href="#">Logout</a>
+        </nav>
```
Comment (emmiiorji) [inline about.html]: [Optional] For a better user experience on the website, I recommend you modify the page to remove the horizontal scroll.

Your design:
image

image

File: about.html [hunk 16]
```diff
@@ -0,0 +1,153 @@
...
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner3.png" alt="" />
+              </div>
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner4.png" alt="" />
+              </div>
+              <div class="partners-list">
+                <img class="partner-image" src="https://raw.githubusercontent.com/tashisrael/Capstone1/985f739e393fa40af073b99c2f8c89afa7a4e071/images/partner5.png" alt="" />
+              </div>
+            </div>
+          </section> -->
```
Comment (emmiiorji) [inline about.html]: As suggested above, kindly avoid leaving pieces of commented code in your code base. The commit history can always take us back in time.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Nowembabazi, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost there

 ## Good Highlights✔

- Some requested changes have been implemented
- Linter checks are passing
- Adherence to GitHub flow


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danielufeli — APPROVED** [review]
Hi @Nowembabazi ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Design ✔️

## Optional suggestions
- [Optional] Kindly update your ReadMe 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**